### ISSUE TITLE
Incorrect activation function / loss function for class_loss

### ISSUE THREAD

**Issue by zazeng** [zzh8829/yolov3-tf2#194]
```sparse_categorical_crossentropy``` [requires the output param](https://www.tensorflow.org/api_docs/python/tf/keras/backend/sparse_categorical_crossentropy
) to be:
A tensor resulting from a softmax (unless from_logits is True, in which case output is expected to be the logits).

https://github.com/zzh8829/yolov3-tf2/blob/65294d5dc1794b325db5a37b2ed02773ca5bf839/yolov3_tf2/models.py#L306-L308

The activation function currently being used is sigmoid. https://github.com/zzh8829/yolov3-tf2/blob/65294d5dc1794b325db5a37b2ed02773ca5bf839/yolov3_tf2/models.py#L159 

Changing the above line to tf.softmax works from a tensorflow point of view but removes the "multi label" modelling aspect. Should it be switched to tf.softmax before the implementation of ```binary_crossentropy``` function?

**Comment by ZF4444** [comment]
I agree with you
